paperqa/utils.py

```diff
@@ -519,3 +520,25 @@ def extract_thought(content: str | None) -> str:
     "peer-review": "misc",  # No direct equivalent, so 'misc' is used
     "other": "article",  # Assume an article if we don't know the type
 }
+
+
+@contextlib.contextmanager
+def logging_filters(loggers: set[str], filters: set[type[logging.Filter]]):
```

Comment:
```suggestion
def logging_filters(loggers: Collection[str], filters: Collection[type[logging.Filter]]):
```

Can we do `Collection` or `Sequence` here

Reply:
sure